doc: imgtool: importance of clear flag (#2202)
mcu-tools/mcuboot: docs/encrypted_images.md

would be very hard to determine this information when an interruption
occurs and the information is spread across multiple areas.

## [Factory-programing requirement:](#factory-programing-requirement)

It is important to have updates without any voids in encryption, therefore
from the very begining, flags and TLV's has to set accordingly.
Perform first flashing with encryption settings as if it was meant for DFU

> and add `--clear` flag to keep image unencrypted and redy to go.

Reviewer: Perform the first flashing with image signed by imgtool with encryption settings (...) appended by '--clear` (..)

would be very hard to determine this information when an interruption
occurs and the information is spread across multiple areas.

> ## [Factory-programing requirement:](#factory-programing-requirement)

Reviewer: remove the colon

> It is important to have updates without any voids in encryption, therefore
> from the very begining, flags and TLV's has to set accordingly.
> Perform first flashing with image signed by imgtool with encryption settings
> as if it was meant for DFU, appended by `--clear` flag to keep the image
> unencrypted and redy to go.

Reviewer 2: ```suggestion
It is important to have updates without any voids in encryption. 
Therefore, from the very beginning, flags and TLV's must be set accordingly.
Perform the first flashing with an image signed by imgtool with encryption settings
intended for DFU.
Append it with the `--clear` flag to keep the image unencrypted and ready for use.
```
